lib/stringify.js

```diff
@@ -20,14 +20,20 @@ var defaults = {
     delimiter: '&',
     encode: true,
     encoder: utils.encode,
+    format: 'RFC3986',
+    formatters: {
+        /** @return {string} */
```

Comment:
How could IDE parse the types of the variables, for example, when JavaScript doesn't have strict types?